package.json

```diff
@@ -1459,6 +1460,7 @@
     "@types/redux-actions": "^2.6.1",
     "@types/redux-logger": "^3.0.8",
     "@types/resolve": "^1.20.1",
+    "@types/rusha": "^0.8.3",
```

Comment:
Thx for taking care of this so quickly 👍 QQ: why not using `kbn-crypto-browser`, it's not sha1 but sha256, which should do the job, and no new dependency would be necessary (and it just this could be used, to simplify the code)
https://github.com/elastic/kibana/blob/4824d9da8c59f03522779d1b41d531f52b255dab/packages/kbn-crypto-browser/README.md#L4-L3

Reply:
good idea here :)